.env

```
@@ -3,7 +3,7 @@ DOCKER_CLIENT_TIMEOUT=320
 COMPOSE_HTTP_TIMEOUT=320
 COMPOSE_PROJECT_NAME=po_off
 COMPOSE_PATH_SEPARATOR=;
-COMPOSE_FILE=docker-compose.yml;docker/dev.yml
+COMPOSE_FILE=docker-compose.yml;docker/dev.yml;docker/run.yml
```

```suggestion
COMPOSE_FILE_BARE=docker-compose.yml;docker/dev.yml
COMPOSE_FILE=${COMPOSE_FILE_BARE};docker/run.yml
```
Removing the need for the complexity in line 49 of the Makefile

---

Also wondering if this should be called COMPOSE_FILE_RAW to be consistent with the naming [here](https://github.com/openfoodfacts/.github/blob/main/docs/general-project-conventions.md#docker-compose)